posthog-node/src/types.ts

```diff
@@ -54,7 +54,7 @@ export type PostHogFeatureFlag = {
         rollout_percentage: number
       }[]
     }
-    payloads?: Record<string, JsonType>
+    payloads?: Record<string, string>
```

Comment:
`PostHogCoreOptions.bootstrap.featureFlagPayloads` is still `Record<string, JsonType>`, is the current code working correctly for bootstrapped flags passing `Record<string, JsonType>`? or should we update it there too?

Reply:
Maybe `getAllFlagsAndPayloads` signature should change as well?

Reply:
Yep the bootstrapped ones are the parsed ones, so the type is ok and works as you'd expect.

`getAllFlagsAndPayloads` returns JsonType because this is parsed.

The type I'm updating references what we get from local eval calls, which is not a parsed payload.